src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,42 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
+# Documentation for ORDO commands in this section is in: `docs/developer/ordo.md`
+python-install-dependencies:
+	python3 -m pip install --upgrade pip
+	python3 -m pip install -r $(RELEASEDIR)/requirements.txt
 
-report-mapping-annotations:
+report-mapping-annotations: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
-update-jinja-sparql-queries:
+update-jinja-sparql-queries: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+# TODO: Move away from .PHONY targets
+# TODO: need a purl for this?
+# TODO: use this target: tmp/mondo.sssom.tsv
+.PHONY: mondo-mappings
+MONDO_MAPPINGS_TSV = 'https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv'
+mondo-mappings:
+	wget ${MONDO_MAPPINGS_TSV} -P ../analysis/excluded_terms_in_mondo/ignored/mappings/
+
+# TODO: Need to add dependency for sig/mirror files?
+.PHONY: excluded_terms_in_mondo
+# TODO: can put two file names as the target names
+# excluded-terms-in-mondo: mondo-mappings
+#     'outpath_xrefed_terms': os.path.join(PROJECT_DIR, 'excluded_terms_in_mondo - xrefs.csv'),
+#     'outpath_in_terms': os.path.join(THIS_SCRIPT_DIR, 'ignored', 'output', 'excluded_terms_in_mondo - in_mondo.csv'),
+# TODO: update my script to accept this positional param $@
+# TODO: change from CSV to TSV output
+# note: why not use $@: will only pick the calling target (e.g. if someone calls for the first tone)
+$(REPORTDIR)/excluded_terms_in_mondo_xrefs.csv $(REPORTDIR)/excluded_terms_in_mondo.csv: $(TMPDIR)/mondo.sssom.tsv python-install-dependencies
```

Comment:
@matentzn I used a reference to `$(TMPDIR)` here, but I see that many other make goals are using `tmp/` instead. Just wondering if those will also be changed to `$(TMPDIR)`?

Reply:
`$(TMPDIR)` always, tmp/ is sloppy on my part